### PR TITLE
foreman 2.2: Fix service name in upgrade docs

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: _includes/manuals/2.2/3.6_upgrade.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ For more information about how to backup your instance head over to
 #### Step 2 - Perform the upgrade
 
 Before proceeding, it is necessary to shutdown the Foreman instance (e.g.
-`service httpd stop` or `service apache2 stop` usually).
+`service foreman stop` or `service foreman stop` usually).
```
Comment (bastelfreak) [inline _includes/manuals/2.2/3.6_upgrade.md]: I'm not sure if it's still supported to run foreman through apache. If that's the case I can update the wording.

Comment (ekohl) [reply _includes/manuals/2.2/3.6_upgrade.md]: It is. Note that we're also working on systemd socket activation in which case `foreman.socket` should also be stopped. Perhaps it should be changed to `systemctl stop httpd foreman.service foreman.socket`?

Comment (bastelfreak) [reply _includes/manuals/2.2/3.6_upgrade.md]: updated it. It now assumes systemctl is available. I'm not sure if foreman still supports distributions so old that they don't have systemd, but I would also assume listing `service` isn't required for those rare cases.

Comment (ekohl) [reply _includes/manuals/2.2/3.6_upgrade.md]: No, we're moving away from those.

File: _includes/manuals/2.2/3.6_upgrade.md [hunk 2]
```diff
@@ -32,7 +32,8 @@ For more information about how to backup your instance head over to
 #### Step 2 - Perform the upgrade
 
 Before proceeding, it is necessary to shutdown the Foreman instance (e.g.
-`service httpd stop` or `service apache2 stop` usually).
+`systemctl stop httpd foreman.service foreman.socket` or
+`systemctl apache foreman.service foreman.socket` usually).
```
Comment (tbrisker) [inline _includes/manuals/2.2/3.6_upgrade.md]: missed the stop here
```suggestion
`systemctl stop apache foreman.service foreman.socket` usually).
```

**Review by tbrisker — CHANGES_REQUESTED** [review]
Thanks @bastelfreak, could you also apply the same change to the 2.3 and nightly manuals?

**Comment by bastelfreak** [comment]
@tbrisker I hope I hit all occurrences.

**Review by tbrisker — APPROVED** [review]
Thanks @bastelfreak !

**Pull request merged**